src/book/03-guides/04-general/02-troubleshooting.mdx

```diff
@@ -35,6 +35,31 @@ Run `./b configure --clean`. This refreshes the docker volumes, removing the off
 
 If you have just added a new file to the codebase, make sure that it's part of a role. After you have added it to a role, run `./b configure`, and the build system should detect the new file and compile it the next time you run `./b build`.
 
+### Why am I having CMakeCache errors when I do `./b run <role>`?
```

Comment:
Having a `CMakeCache.txt` file in the NUbots repo can also cause bizarre issues with `./b install` as well. There may also be other issues that this can cause. It might be good to add a note letting people know that if they are getting weird errors with other b-script modules that looking for the `CMakeCache.txt` file may potentially resolve it